test/e2e/operator/operator.go

```diff
@@ -54,4 +59,34 @@ func describe() {
 		err = utils.TestWebhookServerTLS(ctx, f, "https://inteldeviceplugins-webhook-service")
 		gomega.Expect(err).To(gomega.BeNil())
 	})
+
+	ginkgo.It("deploys IAA plugin with operator", func(ctx context.Context) {
+		testPluginWithOperator("iaa", []v1.ResourceName{"iaa.intel.com/wq-user-dedicated"}, f, ctx)
```

Comment:
Is it possible that there is a race condition if the operator tests are run in parallel with the device specific tests?

Reply:
#1482